src/fastoad/openmdao/problem.py

```diff
@@ -38,14 +38,24 @@
 
 
 class FASTOADProblem(om.Problem):
-    """Vanilla OpenMDAO Problem except that it can write its outputs to a file.
+    """
+    Vanilla OpenMDAO Problem except that it can write its outputs to a file.
 
     It also runs :class:`~fastoad.openmdao.validity_checker.ValidityDomainChecker`
     after each :meth:`run_model` or :meth:`run_driver`
     (but it does nothing if no check has been registered).
     """
 
     def __init__(self, *args, **kwargs):
+        if (
+            version.parse(openmdao.__version__) >= version.parse("3.17")
+            and "OPENMDAO_REPORTS" not in os.environ
+            and "reports" not in kwargs
+            and len(args) < 5
+        ):
+            # Automatic reports are deactivated for FAST-OAD, unless OPENMDAO_REPORTS env
+            # variable is set.
```

Comment:
Would it not be useful to handle this option via kwargs "reports" that could be specified in the configuration file or via `_run_problem(reports=True)? Maybe later?

Reply:
Perfect! Then lets stick to this for the moment.